pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -392,6 +395,53 @@ func fixLibvirtCrcNetworkActive() (bool, error) {
 	return true, nil
 }
 
+func checkLibvirtCrcBridgePermissions() (bool, error) {
+	logging.Debug("Checking if 'crc' bridge has appropriate permissions setup")
+	path := qemuBridgeConfigPath
+	_, err := os.Stat(path)
+	if err != nil {
+		logging.Debug(fmt.Sprintf("Failed to open %s: %s, trying %s..", qemuBridgeConfigPath, err, qemuBridgeConfigPathRHEL))
+		path = qemuBridgeConfigPathRHEL
```

Comment:
staggered or nested if's for checking are not the right approach.

```
paths ;= { qemuBridgeConfigPath, qemuBridgeConfigPathRHEL }
for path, _ := range paths {
        _, err := os.Stat(path)
        if err != nil {
            //logging ...
            //continue
        } else {
            // do your thing
            //break / return
        }
    }
```

Reply:
You've the final say but I think what needs to be prevented is nesting in general and in this particular case:

1. with your suggestion, we'll have the same level of nesting.
1. There is only 2 levels of testing (so not bad at all).